views/volunteer.handlebars

```diff
@@ -2,31 +2,117 @@
   {{> top_nav }}
 
   <div class="container mt-5">
+    <div class="row">
+      <div class="col">
+        <h1>
```

Comment:
can you i18n everything? Or are you just getting feedback first and then will follow up?

Reply:
I wasn't planning on translating these pages now since we currently require working knowledge of English to contribute to the project. Do you think we are setup to work with volunteers who can't read / write basic English?